content/api/cli.md

```diff
@@ -1,8 +1,264 @@
 ---
-title: Using the CLI
+title: CLI
 ---
-> Cli call
-> Cli options
-> --config
+## Overview
 
-> see also [[Using the configuration]]
+`webpack` provides a command line interface(cli) to configure and interact with your build.
+This is mostly useful in case of early prototyping, profiling, writing npm scripts or personal customization of the build.
+For proper usage and easy distribution of this configuration, webpack can be configured with `webpack.config.js`.
+Any parameters sent to the CLI will map to a corresponding parameter in the config file.
+
+## Installation
+
+``` sh
+$ npm install webpack -g
+```
+
+The `webpack` command is now available globally.
+
+### Common Usage
+
+``` sh
+webpack <entry> [<entry>] <output>
+```
+| Parameter | Configuration Mapping         | Explanation                                                                                                                                                             |
+|-----------|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| entry     | entry                         | A filename or a set of named filenames which act as the entry point to build your project. If you pass a pair in the form of `=` you can create an additional entry point. |
+| output    | output.path + output.filename | A path and filename for the bundled file to be saved in.                                                                                                                |
+
+#### Examples
+
+If your project structure is as follows -
+```
+.
+├── dist
+├── index.html
+└── src
+    ├── index.js
+    ├── index2.js
+    └── others.js
+```
+
+`webpack src/index.js dist/bundle.js`
+	This will bundle your source code with entry as `index.js` and the output bundle file will have a path of `dist` and the filename will be `bundle.js`
+
+	| Asset     | Size    | Chunks      | Chunk Names |
+	|-----------|---------|-------------|-------------|
+	| bundle.js | 1.54 kB | 0 [emitted] | index       |
+		[0] ./src/index.js 51 bytes {0} [built]
```

Comment:
Yeah. I am not sure how to format webpack output. We must be showing this output in other places. Can you point me to any such instance?